tests/run-multitests.py

```diff
@@ -378,6 +407,15 @@ def run_test_on_instances(test_file, num_instances, instances):
                 last_read_time[idx] = time.time()
                 if out is not None and not any(m in out for m in IGNORE_OUTPUT_MATCHES):
                     trace_instance_output(idx, out)
+                    if out.startswith("WAIT_FOR_REBOOT"):
+                        _, resume, delay_ms = out.split(" ")
```

Comment:
the resume string should be a full name, eg `instance0_resume`.  Then verify that the prefix `instance0` matches what's expected.